pkg/sensors/handler.go

```diff
@@ -251,15 +250,11 @@ func (h *handler) addSensor(op *sensorAdd) error {
 	return nil
 }
 
-func removeAllSensors(h *handler) error {
-	var errs error
+func removeAllSensors(h *handler) {
 	for _, col := range h.collections {
-		if err := col.unload(); err != nil {
-			errs = errors.Join(errs, err)
-		}
+		col.destroy()
```

Comment:
CC: @mtardy

Reply:
yep indeed this is a miss on my part thanks. The destroy hook was added in contrast to the unload one that was used when `enabling/disabling` vs removing completely the sensor. Allowing us to clean up stuff only when it's really removed and not only disabled.

Reply:
Sounds good. Can you please ACK the PR if you agree with the change?

Reply:
Too late but yes I agreed with the change